docs/docRoot.js

```diff
@@ -0,0 +1,230 @@
+            'get': getUserListings,
+        },
+        '/users/{id}/loginDevices': {
+            'post': getLoginDevicesSwagger,
+            'delete': deleteLoggedInDevicesSwagger,
+        },
+        '/users/{id}/imageUpload': {
+            'post': imageUpoadSwagger,
+        },
+        '/users/{id}/imageDelete': {
+            'delete': deleteUserProfilePicSwagger,
+        },
+        '/users/{id}/logout': {
+            'post': logoutSwagger,
+        },
+        '/cities': {
+            'get': getAllCitiesSwagger,
+        },
+        '/cities/{cityId}/listings': {
+            'post': createCityListingSwagger,
+            'get': getAllCityListingsSwagger,
+        },
+        '/cities/{cityId}/listings/{id}': {
+            'get': getCityListingByIdSwagger,
+            'patch': updateCityListingSwagger,
+            'delete': deleteCityListingSchemaSwagger,
+        },
+        '/cities/{cityId}/listings/{id}/imageUpload': {
+            'post': uploadImageSchemaSwagger,
+        },
+        '/cities/{cityId}/listings/{id}/imageDelete': {
+            'delete': deleteImageSchemaSwagger,
+        },
+        '/cities/{cityId}/listings/{id}/pdfUpload': {
+            'post': uploadPDFSwagger,
+        },
+        '/cities/{cityId}/listings/{id}/pdfDelete': {
+            'delete': deletePDFSchemaSwagger,
+        },
+        '/cities/{cityId}/listings/:id/vote': {
+            'post': voteOnListingSwagger,
+        },
+        '/listings': {
+            'get': getAllListingsSwagger,
+        },
+        '/listings/search': {
+            'get': getSearchListingsSwagger,
+        },
+        '/listings/': {
+            'post': createListingSwagger
+        },
+        '/categories': {
+            'get': getAllCategoriesSwagger,
+        },
+        '/categories/{id}/subcategories': {
+            'get': getSubCategoriesSwagger,
+        },
+        '/categories/listingsCount': {
+            'get': getCategoriesListingCountSwagger,
+        },
+        '/status': {
+            'get': getAllStatusesSwagger,
+        },
+        '/citizenServices': {
+            'get': getAllCitizenServicesSwagger,
+        },
+        '/citizenServices/digitalManagement': {
+            'get': getDigitalManagementSwagger,
+        },
+        '/contactUs': {
+            'post': contactUsSwagger,
+        },
+        '/moreinfo': {
+            'get': getMoreInfoSwagger,
+        },
+        '/users/{userId}/favorites': {
+            'get': getFavoritesSwagger,
+            'post': createFavoriteListingSwagger,
+        },
+        '/users/{userId}/favorites/listings': {
+            'get': getFavoriteListingsSwagger,
+        },
+        '/users/{userId}/favorites/{id}': {
+            'delete': deleteFavoriteListingSwagger,
+        },
+        '/cities/{cityId}/villages': {
+            'get': getVillegesSwagger,
+        },
+        '/ads/' :{
+            'get': getAds,
+        },
+        '/cities/:cityId/wasteCalender/streets' : {
```

Comment:
waste calender, defect reporter etc should be available only if the env flag is set. Can you look into those once

Reply:
yeah, okay